test/debug/bp_test.rb

```diff
@@ -24,4 +24,31 @@ def test_breakpoint_fires_correctly
       end
     end
   end
+
+  class BindingBPWithCommandTest < TestCase
+    def program
+      <<~RUBY
+      class Foo
+        def bar
+          binding.bp(command: "continue")
+          baz
+        end
+
+        def baz
+          binding.bp
+        end
+      end
+
+      Foo.new.bar
+      RUBY
+    end
+
+    def test_breakpoint_execute_command_argument_correctly
+      debug_code(program) do
+        type 'continue'
+        assert_line_text(/Foo#baz/)
+        type 'quit'
```

Comment:
I'm sorry, I forgot reviewing this part in https://github.com/ruby/debug/pull/51. To finish debugger correctly, we should use `type y`
```suggestion
        type 'quit'
        type 'y'
```